scripts/md2html.sh

```diff
@@ -15,12 +15,12 @@ cat > "$html_file" <<-EOF
   <link rel="icon" href="{{FAVICON}}" type="image/png"/>
 </head>
 <div class="container-lg px-3 my-5 markdown-body">
-</html>
 EOF
 
 yarn -s remark "$md_file" --use remark-html >> "$html_file"
 
 cat >> "$html_file" <<-EOF
 </div>
 </body>
```

Comment:
Sorry for my carelessness but there is also a missing opening `<body>` tag

Reply:
Fixed.